develop/tutorials/articles/180-themes-and-layout-templates/01-themes/01-creating-themes/00-creating-themes-intro.markdown

```diff
@@ -12,30 +12,37 @@ Follow these steps to install the Liferay Theme Generator and generate a theme:
     generator. 
 
 2.  Use npm to install 
-    [Yeoman](http://yeoman.io/), 
-    [gulp](https://gulpjs.com/), 
-    and the Liferay Theme Generator:
-
-        npm install -g yo gulp generator-liferay-theme
- 
-    If you're on Windows, follow the instructions in step 2a to install Sass, 
-    otherwise you can skip to step 3.
-
-    a.  To use Sass on Windows, you must use either Sass from node-sass or Sass 
-        from Ruby. By default, the generator uses node-sass, but this also 
-        requires that you have Visual Studio on Windows, along with 
-        [node-gyp and Python installed](https://github.com/nodejs/node-gyp#installation). 
-        Since Visual Studio is a particularly large dependency, **if you don't 
-        already have it installed**, we recommend you install Ruby Sass instead. 
-        Install Ruby via the 
-        [Ruby installer](http://rubyinstaller.org/), 
-        and then install the compass gem:
-
-        gem install compass
- 
-    The matching sass gem is automatically fetched and installed as well. 
-
-3.  Run the generator and follow the prompts to create your theme:
+    [Yeoman](http://yeoman.io/) 
+    and 
+    [gulp](https://gulpjs.com/):
+
+        npm install -g yo gulp
+
+3.  Use npm to Install the Liferay Theme Generator. 7.x.x versions of the Theme 
+    Generator let you create themes for 6.2 and 7.0. Versions 8.x.x and up let 
+    you create themes for 7.0 and later versions of @product@. To create themes 
+    for **7.1**, run this command:
+   
+        npm install -g generator-liferay-theme
```

Comment:
Hey @mwilliams2014 Could we use the same pattern than here? https://github.com/sez11a/liferay-docs/pull/3733/commits/a3c5c58ccb5ab6f123d1f3bf650447b0d54dd839

Offering the two options for different versions

What do you think?

Reply:
I suppose we can. Since this is for the 7.1 docs, I didn't want to share info on how to build for 6.2 and 7.0. I can say "To create themes for **7.0**, **7.1**, and later versions of @product@, run this command:"...
Do you think we should provide info on 6.2 and 7.0 themes in the 7.1 docs?

Reply:
To keep it consistent, and to avoid any confusion for developers, I'll make it the same.